spec/features/create_mediated_page_request_spec.rb

```diff
@@ -7,9 +7,11 @@
       visit new_mediated_page_path(item_id: '1234', origin: 'SPEC-COLL', origin_location: 'STACKS')
       click_link "I don't have a SUNet ID"
 
+      expect(page).to have_field('Library ID', type: 'text')
       expect(page).to have_field('Name', type: 'text')
       expect(page).to have_field('Email', type: 'email')
       expect(page).to have_css('a', text: '‹ Go back (show the login option)')
+      expect(page.execute_script("$('input#mediated_page_user_attributes_library_id').trigger('focus')"))
```

Comment:
I don't think this is doing what you're trying to accomplish.  RSpec's [`expect`](https://www.relishapp.com/rspec/rspec-expectations/docs/built-in-matchers) requires that you assert something.  Here, you just have `expect(...)` where you would need `expect(...).to` **or** `expect(...).not_to`.  Simply calling `expect(...)` doesn't actually do anything (besides executing the code passed in).

Also, the `execute_script` here is not what you want.  That is causing the javascript you pass to be executed on the page, effectively causing the referenced input to have the focus event triggered by the test itself. See [`trigger()`](https://api.jquery.com/trigger/).

CSS already has a [`:focus`](http://www.w3schools.com/cssref/sel_focus.asp) selector that we can take advantage of here so we should be able to make an assertion like `expect(page).to have_css('input#mediated_page_user_attributes_library_id:focus')`